src/parsers.ts

```diff
@@ -0,0 +1,110 @@
+import { xdr, SorobanDataBuilder } from 'stellar-base';
+import { SorobanRpc } from './soroban_rpc';
+
+export function parseLedgerEntries(
```

Comment:
minor for consideration, on the parsing methods, try a consistent naming pattern like `parseRPC<ResponseType>Response` rather than 'raw' or no prefix, etc.

Reply:
ah, I see 'Raw' is used consistently further down, maybe just this method follows same pattern like  `parseRawLedgerEntries`

Reply:
good call, fixed up! e7ed703